jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecr/actions/PushToRepositoryAction.kt

```diff
@@ -243,25 +246,26 @@ internal class PushToEcrDialog(
                         text = value.name
                     }
                 )
-                    .growPolicy(GrowPolicy.MEDIUM_TEXT)
-                    .withErrorOnApplyIf(message("ecr.dockerfile.configuration.invalid")) { it.selected() == null }
-                    .withErrorOnApplyIf(message("ecr.dockerfile.configuration.invalid_server")) { it.selected()?.serverName == null }
+                    .applyToComponent {
+                        ComboboxSpeedSearch(this)
 
-                // TODO: how do we render both the Docker icon and action items correctly?
-                box.component.apply {
-                    isEditable = true
-                    editor = object : BasicComboBoxEditor.UIResource() {
-                        override fun createEditorComponent(): JTextField {
-                            val textField = ExtendableTextField()
-                            textField.isEditable = false
+                        // TODO: how do we render both the Docker icon and action items correctly?
+                        isEditable = true
```

Comment:
SpeedSearch on this one seemed wonky..probably due to how we mess with the model? So did not add it